las_geoh5/import_las.py

```diff
@@ -416,3 +420,115 @@ def las_to_drillhole(
             new_row = drillhole.surveys[0, :]
             new_row[0] = np.max(depths)
             drillhole.surveys = np.vstack([drillhole.surveys, new_row])
+
+
+def _patch_lasio_reader():
+    """Patch lasio.reader.configure_metadata_patterns to handle edge cases."""
+
+    # TODO: Propose change on lasio to fix possible version issue
+
+    if getattr(lasio.reader, "patched_configure_metadata_patterns", False):

```

Comment:
What does this do?

Reply:
Just to patch only once. Probably not critical but it helped me detect my patch was lost as it kept patching again and again